fgpyo/sequence.py

```diff
@@ -88,9 +99,55 @@ def longest_hp_length(bases: str) -> int:
     return max_hp
 
 
-def gc_content(bases: str) -> float:
-    """Calculates the fraction of G and C bases in a sequence."""
-    if len(bases) == 0:
+def longest_dinucleotide_run_length(bases: str) -> int:
+    """Number of bases in the longest dinucleotide run in a primer.
+
+    A dinucleotide run is when two nucleotides are repeated in tandem. For example,
+    TTGG (length = 4) or AACCAACCAA (length = 10). If there are no such runs, returns 0."""
```

Comment:
Can you add a `Returns` in the docstring please and be even more explicit that it returns the _sequence_ length not the number of repeat units?  E.g.

`returns the number of bases in the longest dinuc repeat (NOT the number of repeat units).`